roles/oc_setup/tasks/main.yml

```diff
@@ -105,4 +105,52 @@
       subjects:
         - kind: User
           name: "admin"
+
+- name: Add SSH keys to cluster nodes at day2
+  when:
+    - oc_ssh_extra_keys_paths is defined
+    - oc_ssh_extra_keys_paths | length > 0
+  block:
+    - name: Check if SSH keys exist
+      ansible.builtin.stat:
+        path: "{{ item }}"
+      loop: "{{ oc_ssh_extra_keys_paths }}"
+      register: _os_ssh_path
+
+    - name: "Fail if any ssh path does not exist"
+      ansible.builtin.fail:
+        msg: "SSH public key {{ key.item }} file does not exist"
+      when: not key.stat.exists
+      loop: "{{ _os_ssh_path.results }}"
+      loop_control:
+        loop_var: key
+        label: "{{ key.item }}"
+
+    - name: Set SSH Keys manifests in master group
```

Comment:
Maybe we can check the available machine pools and loop through them if we have more than masters and workers.

```
    - name: Retrieve MachineConfigPool resources
      community.kubernetes.k8s_info:
        api_version: machineconfiguration.openshift.io/v1
        kind: MachineConfigPool
      register: mcp_info

    - name: Extract MachineConfigPool Names
      set_fact:
        mcp_names: "{{ mcp_info.resources | map(attribute='metadata.name') | list }}"
```